l10n/en/footer-refresh.ftl

```diff
@@ -0,0 +1,41 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+footer-refresh-find-out-about = Find out about { -brand-name-mozilla } products, initiatives, and more. We’ll never sell your email, even if presented with free pizza for life.
+footer-refresh-leadership = Leadership
+footer-refresh-advertise = Advertise with { -brand-name-mozilla }
+footer-refresh-firefox-release-notes = { -brand-name-firefox } Release Notes
+footer-refresh-mdn = MDN
+footer-refresh-follow-mozilla = Follow @{ -brand-name-mozilla }
+footer-refresh-x = X
+footer-refresh-x-formerly-twitter = X (formerly Twitter)
+footer-refresh-mastodon = { -brand-name-mastodon }
+footer-refresh-instagram = { -brand-name-instagram }
+footer-refresh-linkedin = { -brand-name-linkedin }
+footer-refresh-tiktok = { -brand-name-tiktok }
+footer-refresh-spotify = { -brand-name-spotify }
+footer-refresh-follow-firefox = Follow @{ -brand-name-firefox }
+footer-refresh-youtube = { -brand-name-youtube }
+footer-refresh-company = Company
+footer-refresh-press-center = Press Center
+footer-refresh-careers = Careers
+footer-refresh-contact = Contact
+footer-refresh-support = Support
+footer-refresh-product-help = Product Help
+footer-refresh-file-a-bug = File a Bug
+footer-refresh-localize-mozilla = Localize { -brand-name-mozilla }
+footer-refresh-resources = Resources
+footer-refresh-brand-standards = Brand Standards
+footer-refresh-developers = Developers
+footer-refresh-developer-edition = { -brand-name-developer-edition }
+footer-refresh-enterprise = { -brand-name-enterprise }
+footer-refresh-tools = Tools
+footer-refresh-donate = Donate
+footer-refresh-visit-mozilla-corporations = Visit <a { $moco_link }>{ -brand-name-mozilla-corporation }’s</a> not-for-profit parent, the <a { $mofo_link }>{ -brand-name-mozilla-foundation }</a>.
+footer-refresh-portions-of-this-content = Portions of this content are ©1998–{ $current_year } by individual mozilla.org contributors. Content available under a <a rel="license" href="{ $url }">{ -brand-name-creative-commons } license</a>.
```

Comment:
**Nit:** we should try and avoid `<a rel="license" href="{ $url }">` in favor of `<a rel="license" { $href }>` like elsewhere here.